drgn/helpers/memory.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+Memory
+------
+
+The ``drgn.helpers.memory`` module provides helpers for working with memory and addresses.
+"""
+
+import operator
+from typing import Optional, Union, overload
+
+import drgn
+from drgn import IntegerLike, Object, Program, SymbolKind
+from drgn.helpers import escape_ascii_string
+from drgn.helpers.linux.slab import find_containing_slab_cache
+
+__all__ = ("identify_address",)
+
+
+SYMBOL_KIND_STR = {
+    SymbolKind.UNKNOWN: "symbol",
+    SymbolKind.OBJECT: "symbol object",
```

Comment:
`object symbol` reads more naturally to me than `symbol object`. Can you swap the wording for all of the kinds?

Reply:
Fixed.